translation/xliff.rst

```diff
@@ -0,0 +1,44 @@
+The XLIFF format
```

Comment:
I know this article explains something very specific ... but creating a new tiny article of just 44 lines is what we're trying to avoid recently. Couldn't we add this in a section (or sidebar) of the main translation article?

Reply:
I somewhat agree, but I don't think this matches our 80-20 rule. XLIFF formats have some specialized features, notes being one that is supported by Symfony. I don't think XLIFF notes belong to the 80% use cases of translation in a Symfony application. And if they do, it would target the translator more than a PHP developer (as notes are just for usage in the XLIFF editor AFAICS).